modules/openshift-developer-cli-debugging-an-application.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/debugging-applications-in-odo.adoc
+
+[id="debugging-an-application_{context}"]
+
+= Debugging an application
+
+.Procedure
+
+. After an application is deployed, to debug the application start the port forwarding for your component:
++
+----
+$ odo debug port-forward
+----
+
+. Attach the debugger bundled in your IDE to the component as per instructions relevant to your IDE.
```

Comment:
Suggest "Attach the debugger bundled in your IDE to the component. See the documentation for your IDE for instructions." As long as you think it's probably documented. And if not, can always do something like "Attach the debugger bundled in your IDE to the component. Instructions vary depending on your IDE."